src/views/states/list/StatesList.tsx

```diff
@@ -95,21 +100,30 @@ const StatesList: FC = () => {
               }
               className="list-managment-group__pagination"
               defaultToFullPage
-              itemCount={filteredData?.length}
+              itemCount={sortedStates?.length}
               page={pagination?.page}
               perPage={pagination?.perPage}
               perPageOptions={paginationDefaultValues}
             />
           </div>
 
-          {filteredData.length > 0 ? (
-            <Table
-              cells={activeColumns}
-              rows={paginatedData}
-              gridBreakPoint={TableGridBreakpoint.none}
-              role="presentation"
-            >
-              <TableHeader />
+          {sortedStates.length > 0 ? (
```

Comment:
Maybe `sortedStates?.length` ? As we use it like that a few lines earlier...

Reply:
Okay